rero_ils/modules/holdings/cli.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Click command-line interface for item record management."""
+
+from __future__ import absolute_import, print_function
+
+import json
+
+import click
+from flask.cli import with_appcontext
+
+from ..documents.api import DocumentsSearch
+from ..holdings.api import create_holding
+from ..item_types.api import ItemTypesSearch
+from ..locations.api import LocationsSearch
+from ..organisations.api import Organisation
+from ..utils import read_json_record
+
+
+def get_documents_with_type_journal():
+    """Get pids of documents with type journal."""
+    document_pids = []
+    es_documents = DocumentsSearch()\
+        .filter('term', type="journal").source(['pid']).scan()
+    return [es_document.pid for es_document in es_documents]
+
+
+def get_location(library_pid):
+    """Rertun a location for the given library pid."""
```

Comment:
Rertun -> Return, but better make the docstring conform: such as 1th line the description then you can user @params and @returns.